pkg/commands/workload.go

```diff
@@ -807,9 +809,9 @@ func (opts *WorkloadOptions) DefineFlags(ctx context.Context, c *cli.Config, cmd
 	cli.NamespaceFlag(ctx, cmd, c, &opts.Namespace)
 	cmd.Flags().StringVarP(&opts.FilePath, cli.StripDash(flags.FilePathFlagName), "f", "", "`file path` containing the description of a single workload, other flags are layered on top of this resource. Use value \"-\" to read from stdin")
 	cmd.Flags().StringVarP(&opts.App, cli.StripDash(flags.AppFlagName), "a", "", "application `name` the workload is a part of")
-	cmd.Flags().StringVarP(&opts.Type, cli.StripDash(flags.TypeFlagName), "t", "", "distinguish workload `type`")
+	cmd.Flags().StringVarP(&opts.Type, cli.StripDash(flags.TypeFlagName), "t", "web", "distinguish workload `type`")
```

Comment:
why dont you use `WebTypeReservedKey` here?

Reply:
changed to use the reserved key